### ISSUE TITLE
Images not working

### ISSUE THREAD

**Issue by hovski** [hovski/ecommerce-pattern-library#4]
@thomasjbradley For some reason I can't get the images to show up on any parts of my website. If you could let me know what's up, that'd be great!

**Comment by thomasjbradley** [comment]
I only see one place where the products collection is being used, that’s on `products.html`. And there are two small typos that a preventing it from working:

https://github.com/hovski/ecommerce-pattern-library/blob/master/products.html#L12
`site.products` — a dot, not a dash

https://github.com/hovski/ecommerce-pattern-library/blob/master/products.html#L13
`{% pattern cards/main-card data=product %}` — lowercase “p” in product, to match the spelling on the line above.

**Comment by hovski** [comment]
I fixed this. It shows the images now for the products. My only thing is, within my Sections. There are sections that have images on grid format. Those images show as broken image icons not the images themselves and I’m not sure how to resolve this issue.

**Comment by thomasjbradley** [comment]
I don’t see any code within the different tea sections—like what was written on `products.html` that would output the tea cards into those patterns.

**Comment by hovski** [comment]
Yeah, see that’s part of my design problem. Each card is just a static image of any tea type with its hoverstate. The only time teacard content should show similar to the products page would be on the section another student would be designing. My sections are just images of tea. When you click them it goes to /product-description/ page not my products page. I don’t know if I’m making sense.

**Comment by thomasjbradley** [comment]
On the `products.html` page (after you’ve made the above fixes) how is what’s being output at the bottom different than what you want in the different tea colour sections?

**Comment by hovski** [comment]
So the best way to describe my problem is this.

If I were to go to my localhost/products/betty-white/ lets say. The image
shows up, the content for that card shows up. Everything shows up, but not
inside my banner where it should be (which is fine for now, but it is
supposed to be showing up inside one of my banners called
product-description.) My product-description stuff (I've realized is the
page a student would be making, but since one assignment wanted us to do
all the .md things I ended up making content for my cards).

My homepage/index has nothing but 1 banner, and 1 carousel (carousel
content is just an image which should be working but isn't. It used to work
until I made these .md files) And my products page is just a page with
sections of 8 different teas per section, but those images also don't work.

The student does the other page that goes to the item and explains what the
item is. I'm going to attach photos below to explain this better. The image
with the arrow is the page the student would make (and you'll see why I did
the .md's/content for that page since my pages don't have any descriptions
of my products on either index or products.

**Comment by thomasjbradley** [comment]
The photos don’t come through if you attach them to an email response. You have to upload them directly to GitHub.

**Comment by hovski** [comment]
![unnamed](https://user-images.githubusercontent.com/31852182/54725185-8869b880-4b44-11e9-8113-0e52a26ecf6a.jpg)
![12312321](https://user-images.githubusercontent.com/31852182/54725189-8d2e6c80-4b44-11e9-9248-6f5155fe9920.jpg)

**Comment by thomasjbradley** [comment]
Yes, I completely understand all that.

What I’m trying to say is that your products page tea secions don’t work **because you haven’t written any code yet**! If you look currently at the `products.html` file and view it in the browser, what’s outputting at the bottom of that page? *All the products from the MD files*—**and only their image**.

The images within the tea sections don’t work because you haven’t used the code you wrote in the `products.html` within each of those tea sections to output the images.

**Comment by hovski** [comment]
Holy moses you're a genius. I didn't even realize this till you told me.
Ooof, thank you.

**Comment by hovski** [comment]
So I'm clearly not getting what you've explained to me. Can I meet up with
you to go over this coding issue?
